### ISSUE TITLE
fix current k8s-mesos test flakes

### ISSUE THREAD

**Issue by jdef** [mesosphere/kubernetes-mesos#432]
- TestPlugin_LifeCycle
  - https://github.com/GoogleCloudPlatform/kubernetes/issues/11901
- TestDQ
  - https://github.com/GoogleCloudPlatform/kubernetes/issues/11857
  - https://github.com/GoogleCloudPlatform/kubernetes/issues/11821

**Comment by jdef** [comment]
possibly needs a rewrite since it's very hard to test; API is timeout based, difficult to test in a heavily loaded CI environment

**Comment by jdef** [comment]
did they not show up because the tests were disabled?

**Comment by sttts** [comment]
Yes, both tests are disabled for quite some time. So we don't know.

**Comment by jdef** [comment]
The code hasn't really changed since the tests were disabled, so I'm pretty
confident that the test flakes are still there

**Comment by karlkfi** [comment]
I have a branch and local code with changes, but it became a giant hairy mess, and I had more important things to get to. So I put it on hold.
